log.go

```diff
@@ -0,0 +1,212 @@
+package stripe
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+//
+// Public constants
+//
+
+const (
+	LevelError Level = 1
+	LevelWarn Level = 2
+	LevelInfo Level = 3
+	LevelDebug Level = 4
+
+	// Older deprecated levels for Printfer-style logging.
+	printferLevelError = 1
```

Comment:
These are new constants for a deprecated idea, but they're not exported, and make code much more readable.